woocommerce/woocommerce
# Update Composer to v2 (includes woocommerce-blocks 3.8.0 and woocommerce-admin 1.7.0)

### All Submissions






### Changes proposed in this Pull Request:

This PR updates the version constraint of the `composer/installers` dependency so that it is more permissive. In order to facilitate this we've also had to make the constraint more permissive in WooCommerce Blocks and WooCommerce Admin, so this PR includes updates to those two packages. If you could leave the appropriate merge PR information as a comment here that would be very helpful!

Closes #28122.

### How to test the changes in this Pull Request:

1. Run `composer install` after updating to Composer v2
2. Ensure all tests pass!

### Other information:




### Changelog entry

> Dev - Updated to Composer v2
